### ISSUE TITLE
receive hangs and times out for auth number 

### ISSUE THREAD

**Issue by developersteve** [bbernhard/signal-cli-rest-api#429]
### The problem

Completely fine sending messages but unable to receive any using the same authenticated number. The receive endpoint just hangs then throws a timeout, Nothing showing in logs and no issues seem to be appearing. 

Wondering if this is possibly to do with the mode and i should be using `json-rpc` mode? 

### Are you using the latest released version?

- [X] Yes

### Have you read the troubleshooting page?

- [X] Yes

### What type of installation are you running?

signal-cli-rest-api Docker Container

### In which mode are you using the docker container?

Native Mode

### What's the architecture of your host system?

x86-64

**Comment by bbernhard** [comment]
Please enable the debug logging and try to run the `signal-cli` commands directly (see https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/DEBUG.md). This helps to determine whether this is an issue with [signal-cli](https://github.com/AsamK/signal-cli) or the REST API wrapper.

**Comment by developersteve** [comment]
@bbernhard yup did that, same result with the `/v1/receive` endpoint GET call just hanging and timing out. 

<img width="599" alt="image" src="https://github.com/bbernhard/signal-cli-rest-api/assets/1458790/34fad4be-aafc-4ca2-8018-1cf7cbe6f55b">

**Comment by bbernhard** [comment]
Please try with the latest version (0.80)

**Comment by steveredden** [comment]
![image](https://github.com/bbernhard/signal-cli-rest-api/assets/35814432/429890a1-958d-46d3-8b4a-79980348d9c0)

same - i've tried a number of things.

it does succeed from the CLI, but takes >60 sec

![image](https://github.com/bbernhard/signal-cli-rest-api/assets/35814432/ddd83089-738b-4357-a7b2-41d1f2a13e1c)

**Comment by bbernhard** [comment]
Closing this old ticket. If that's still an issue, please create a new one.
